### DISCUSSION TITLE
UEFI HTTP boot

### DISCUSSION THREAD

**Discussion by nanderer** [antonym/netboot.xyz#1066] [Ideas]
Hi folks,

right now i am trying to get used to UEFI, because using the Legacy/CSM mode, like i do much to often, of modern systems to boot my live systems cant stay the answer forever...

Thereby i found out that UEFI has an integrated HTTP boot, so it might be possible to boot over netboot.xyz without any local (disk/flash) image or (PXE) Server...

here are some screenshots:
![image](https://user-images.githubusercontent.com/45161625/148669193-edbb0fd4-ca1c-4230-ac3b-ae512f2456f9.png) 
![image](https://user-images.githubusercontent.com/45161625/148669245-a9035442-84a7-4d24-b02f-fb72928507e0.png)
 ![image](https://user-images.githubusercontent.com/45161625/148669226-ebab08d2-42ad-4f00-85ad-c729835e9f0d.png)
 
so would it be possible to boot some live systems just by connecting a lan cable and enter an url like netboot.xyz/uefi in your uefi firmware menu?

thanks a lot in advance

**Comment by antonym** [comment]
You should be able to use either https://boot.netboot.xyz/ipxe/netboot.xyz.efi or https://boot.netboot.xyz/ipxe/netboot.xyz-snp.efi as a URI for EFI.
